src/components/Modals/WorldPermissionsModal/ModelTabs/Layouts/WorldPermissionsAvatarWithInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { AvatarFace } from 'decentraland-ui/dist/components/AvatarFace/AvatarFace'
+import LoadingText from 'decentraland-ui/dist/components/Loader/LoadingText'
+import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
+import { getResumedAddress } from '../../utils'
+import { WorldPermissionsAvatarWithInfoProps } from './WorldPermissionsAvatarWithInfo.types'
+
+import styles from './WorldPermissionsAvatarWithInfo.module.css'
+
+export default React.memo(function WorldPermissionsAvatarWithInfo(props: WorldPermissionsAvatarWithInfoProps) {
+  const { wallet, profiles, loading } = props
+
+  if (loading || !profiles || !wallet) {
```

Comment:
I think it will be better to make a container for this component. The component should receive only the address of the user and the container should get the profile from the state and if it's loading or not (both selectors are available as the profile module in the `decentraland-dapps` repo)